libs/sdk-ui-tests/scenarios/charts/waterfallChart/coloring.tsx

```diff
@@ -0,0 +1,59 @@
+// (C) 2007-2019 GoodData Corporation
+import { WaterfallChart, IWaterfallChartProps } from "@gooddata/sdk-ui-charts";
+import { IMappingHeader } from "@gooddata/sdk-ui";
+import { isColorDescriptor } from "@gooddata/sdk-model";
+
+import { scenariosFor } from "../../../src";
+import { BlackColor, CustomColorPalette, CustomPaletteColor, RedColor } from "../../_infra/colors";
+import { coloringCustomizer } from "../_infra/coloringVariants";
+import { WaterfallChartWithMultiMeasuresAndViewBy, WaterfallChartWithSingleMeasureAndViewBy } from "./base";
+import { ScenarioGroupNames } from "../_infra/groupNames";
+
+const TotalPredicate = (header: IMappingHeader) => {
+    return isColorDescriptor(header) && header.colorHeaderItem.id.includes("total");
+};
+const PositivePredicate = (header: IMappingHeader) => {
+    return isColorDescriptor(header) && header.colorHeaderItem.id.includes("positive");
+};
+const NegativePredicate = (header: IMappingHeader) => {
+    return isColorDescriptor(header) && header.colorHeaderItem.id.includes("negative");
+};
+
+const colorsAndPalette = scenariosFor<IWaterfallChartProps>("WaterfallChart", WaterfallChart)
+    .withGroupNames(...ScenarioGroupNames.Coloring)
+    .withVisualTestConfig({ groupUnder: "coloring" })
+    .withDefaultTags("vis-config-only", "mock-no-scenario-meta")
+    .addScenarios("coloring", WaterfallChartWithSingleMeasureAndViewBy, coloringCustomizer);
+
+const colorAssignment = scenariosFor<IWaterfallChartProps>("WaterfallChart", WaterfallChart)
+    .withGroupNames(...ScenarioGroupNames.Coloring)
+    .withDefaultTags("vis-config-only", "mock-no-scenario-meta")
+    .addScenario(
+        "assign color to measures",
+        {
+            ...WaterfallChartWithMultiMeasuresAndViewBy,
+            config: {
+                colorPalette: CustomColorPalette,
+                colorMapping: [
+                    {
+                        predicate: TotalPredicate,
+                        color: BlackColor,
+                    },
+                    {
+                        predicate: NegativePredicate,
+                        color: RedColor,
+                    },
+                    {
+                        predicate: PositivePredicate,
+                        color: CustomPaletteColor,
+                    },
+                ],
+                total: {
+                    enabled: true,
```

Comment:
I'm not sure that it's the same storybook screenshot, but please check https://github.com/gooddata/gooddata-ui-sdk/pull/3733/files#diff-3fcf1c3545a7e59039790fb462c7db8304a9c1e406a1a947cdd055586f12177c there's no total visible in that image and it's enabled: true here.